resources/js/components/open/forms/OpenFormField.vue

```diff
@@ -1,21 +1,30 @@
 <template>
-  <div v-if="!isFieldHidden"
-       :id="'block-'+field.id" :class="getFieldWidthClasses(field)"
+  <div
```

Comment:
Please revert the changed here, and only keep the one relevant to the phone input change